example/Example.res

```diff
@@ -91,7 +95,21 @@ module Form = {
         render={({field: {name, onBlur, onChange, ref, value}}) =>
           <div>
             <label> {name->React.string} </label>
-            <input name onBlur onChange ref value />
+            <input
+              name onBlur={_event => onBlur()} onChange={Controller.handleEvent(onChange)} ref value
+            />
+            <button
+              type_="button"
+              onClick={_event =>
+                onChange(
+                  Controller.OnChangeArg.value(
+                    Js.Json.string(
+                      value->Js.String2.split("")->Js.Array2.reverseInPlace->Js.Array2.joinWith(""),
+                    ),
+                  ),
```

Comment:
Look ma, no `setValue`!

Reply:
it will reduce some hard time for ATS, of course I do know you dont really like this api from react-hook-form

Reply:
Not a big fan of how inconsistent the code is 😕 
I would have loved to receive 2 different functions instead of a mysterious one XD

Reply:
agreed ! We hard to know the behavior of this one exactly if we dont check the implementation ( based on the doc, it just said event is any =)))